packages/server/src/tests/utilities/TestConfiguration.ts

```diff
@@ -867,23 +868,10 @@ export default class TestConfiguration {
 
   // QUERY
 
-  async previewQuery(
-    request: any,
-    config: any,
-    datasource: any,
-    fields: any,
-    params: any,
-    verb?: string
-  ) {
+  async previewQuery(request: any, config: any, queryPreview: QueryPreview) {
```

Comment:
Huh. Sorry, I had no idea this was here. When I made my comment I was thinking of something like `config.api.query`, the `QueryAPI` class. I wasn't clear on that at all, my bad. Would you mind moving this to there, for consistency?

Reply:
No worries - moved!